utils/paste.tsx

```diff
@@ -32,6 +35,15 @@ export function getTable(clipboardData: DataTransfer): HTMLTableElement | null {
     return table;
 }
 
+export function getHasLinks(clipboardData: DataTransfer): boolean {
+    if (Array.from(clipboardData.types).indexOf('text/html') === -1) {
+        return false;
+    }
+
+    const html = clipboardData.getData('text/html');
+    return (/<a/i).test(html);
+}
```

Comment:
can be simplified a bit:

```
return Array.from(clipboardData.types).indexOf('text/html') > -1 && (/<a/i).test(clipboardData.getData('text/html'));

Reply:
If we're changing that, you can also change the `.indexOf('text/html') > -1` to `.includes('text/html)`.